util/xxhash.cc

```diff
@@ -476,4 +554,425 @@ U32 XXH32_digest (void* state_in)
     return h32;
 }
 
+/* *******************************************************************
+*  64-bit hash functions
+*********************************************************************/
+
+#if defined(_MSC_VER)     /* Visual Studio */
+#  define XXH_swap64 _byteswap_uint64
```

Comment:
I guess you actually want to define `XXH_read64()` here,
instead of `XXH_swap64()` which is re-defined later on.